real/settings.yml

```diff
@@ -8,14 +8,14 @@ functions:
         if:
           - check: 'world-name'
             description: 'A list of world names which mobs cannot be levelled in.'
-            is-in-list: []
             case-sensitivity: true
+            excluded: []
```

Comment:
Make sure all lists end up at the bottom to prevent clutter if for any reason this gets switched over to the extended syntax. Also, I flipped this and the `mob-owner` list because this one is exclusive, not the `mob-owner` one.

Reply:
Hi Artel - thank you very much for the PR. I currently disagree with the use of the terms 'included' and 'excluded' (see Discord reply).

However, I would be glad to merge in your change where the lists are at the bottom to prevent clutter.

None of the lists should have been inverted: they are meant to be inclusive/exclusive as they were set. I can explain this if you like.